ppss.yaml

```diff
@@ -68,8 +68,9 @@ trader_ss:
 
 sim_ss: # sim only
   log_dir: logs
-  test_n: 5000 # number of epochs to simulate
+  test_n: 1800 # number of epochs to simulate
```

Comment:
yes, I lowered the number of epochs to simulate because fetching the lake data for the initial value takes a lot of time, so this will reduce that and allow the fetch of initial data over 10 days period instead of 30 days